### DISCUSSION TITLE
The Mordor map

### DISCUSSION THREAD

**Discussion by FabioOG** [redeclipse/base#1144] [Maps and Content Showcase]
A big map, open with some small buildings and mountains, designed for sniping, and appropriate for bomber-ball.
Warning : A bit unappropriated do deathmatch because bots often fall in lava or cut themselves.
Description : "3000 years after, Sauron rebirths in omega team, and fight against the alpha team"
![image](https://user-images.githubusercontent.com/35398459/35480684-061f9ba6-0414-11e8-8b81-7f281403c922.png)
![image](https://user-images.githubusercontent.com/35398459/35480689-1bf74e38-0414-11e8-81a9-c6cf39f1f555.png)
Note : there is no map music yet

**Comment by FabioOG** [comment]
P.S : [mordor.zip](https://github.com/red-eclipse/world/files/1670955/mordor.zip)

**Comment by cloudziq** [comment]
is this finished map?  not so dark there?

**Comment by acerspyro** [comment]
Note that this map is made for 1.6 rather than 2.0, thus it will not be compatible with future versions of Red Eclipse.

**Comment by FabioOG** [comment]
How can i make it compatible ?

**Comment by acerspyro** [comment]
@Goldenbrain1 Install the dev version (v1.9.9) of Red Eclipse and load your map from there.

You will have to adjust some of the lights.

**Comment by FabioOG** [comment]
I recently tried to update the map but all the textures I used are not available in 1.9.9, and it will take some time to pick new textures. Is it a bug or are the volcano textures i use suppressed ?

**Comment by SniperGoth** [comment]
Wicked's texture pack was taken out because of licensing reasons afaik.

**Comment by FabioOG** [comment]
Oh, isn't there a way to replace the wicked textures by new similar ones ?

**Comment by no-lex** [comment]
Additionally trak4 is also gone because it is low quality (allegedly).

You willl have to replace textures by hand, and will probably have to vcolour it to get a decent enough approximation of the old dark stone textures that were removed. The best stone replacement is now nobiax/79 (1024x) and the best wood replacement for wicked is nieb/woodfloor1 (512x), but both will probably require vcolour to match old textures.

**Comment by qreeves** [comment]
trak4 was just ugly 😛

**Comment by FabioOG** [comment]
Ty ill try that

**Comment by FabioOG** [comment]
Here is mordor2 with better lightning, new re texture and a map border
[mordor2.zip](https://github.com/red-eclipse/world/files/2151815/mordor2.zip)
